exercises/practice/paasio/test_utils.py

```diff
@@ -0,0 +1,146 @@
+import errno
+import inspect
+import io
+import os
+
+
```

Comment:
```suggestion
LOGGER = logging.getLogger(__name__)
```

Reply:
I can't suggest on the main test file here, but if you change the import on line 6 there to be:

`from test_utils import MockSock, MockFile, MockException, ZEN, SuperMock, LOGGER`

and remove  `LOGGER = logging.getLogger(__name__)`

then that should fix it.